.gitignore

```diff
@@ -28,5 +28,6 @@
 /public/assets
 
 node_modules
+.env
 # Ignore master key for decrypting credentials and more.
```

Comment:
![image](https://github.com/MMhaidari/blog-app/assets/93039359/9b00a289-2e00-4f4b-89b8-919e34399e6a)

- [ ] It would be nice if you fix the links highlighted in the image above because right now they are broken 😨 Kindly fix this link or remove them if you don't have any section in your readme for them to make your readme descriptive and professional 👍